src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
+
+import static java.util.Objects.requireNonNull;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.HomeOfficeApi;
+
+
+@Component
+public class HomeOfficeCaseNotificationsHandler implements PreSubmitCallbackHandler<AsylumCase> {
+
+    private final HomeOfficeApi<AsylumCase> homeOfficeApi;
+    private final boolean isHomeOfficeIntegrationEnabled;
+
+    public HomeOfficeCaseNotificationsHandler(
+        @Value("${featureFlag.isHomeOfficeIntegrationEnabled}") boolean isHomeOfficeIntegrationEnabled,
+        HomeOfficeApi<AsylumCase> homeOfficeApi) {
+        this.isHomeOfficeIntegrationEnabled = isHomeOfficeIntegrationEnabled;
+        this.homeOfficeApi = homeOfficeApi;
+    }
+
+    public boolean canHandle(
+        PreSubmitCallbackStage callbackStage,
+        Callback<AsylumCase> callback
+    ) {
+        requireNonNull(callbackStage, "callbackStage must not be null");
+        requireNonNull(callback, "callback must not be null");
+
+        return isHomeOfficeIntegrationEnabled
```

Comment:
Let's try to do this in Launch Darkly. It is recommended by PE now. I think we can still have HO validation flag and F&P ad they epics are ready but for new ones we should stick to Launch Darkly

Reply:
Launch Darkly feature flag added